### ISSUE TITLE
Usage metrics

### ISSUE THREAD

**Issue by aazaff** [paleobot/pbot-dev#35]
A significant deliverable for this program is that we include many different types of metrics to evaluate usage. Some of these are listed in the [NSF Commitments document](https://docs.google.com/document/d/16wS5iGcKSy6OukEUVp7Nr1p1VfnG1RdU/edit?usp=sharing&ouid=114644716048109419747&rtpof=true&sd=true). There seem to be a wide variety of these promised metrics, some of which I don't really understand what was envisioned for them. We will have to discuss these all as a team.

This is **not** an immediate concern, but I wanted to start the issue in GitHub so it doesn't slip through the cracks.

**Comment by clairecleveland** [comment]
Yes, the paleobotany team has this as a rolling "get to it" task item. We plan on having a brainstorm session to start things off. If we don't get to that before you see a need for it, please remind us.

**Comment by clairecleveland** [comment]
I'm adding a note to my calendar for Botany team to get back to this after HackAThon

**Comment by aazaff** [comment]
@NoisyFlowers, do you think we could easily set up google analytics for pbot or something similar?
